sdk/python/feast/feature_view.py

```diff
@@ -263,6 +263,7 @@ def __eq__(self, other):
             or self.batch_source != other.batch_source
             or self.stream_source != other.stream_source
             or sorted(self.entity_columns) != sorted(other.entity_columns)
+            or self.materialization_intervals != other.materialization_intervals
```

Comment:
Without this check, two FeatureViews with identical definitions but created at different times would now be considered "different" because their data sources had different timestamps. Including materialization intervals in equality ensures that the registry correctly identifies when materialization history needs to be preserved, even if it means more registry updates are triggered.

Reply:
> two FeatureViews with identical definitions but created at different times would now be considered "different"

Do feature view name is not enough here to identify them as a `different` ?

Reply:
`__eq__` is not about identifying a unique feature view within a registry. The registry uses equality checks to determine if updates are needed.